### PR TITLE
Fix/fix multilingual fields display

### PR DESCRIPTION
**Pull request by bellisk**

### CODE REVIEW

File: ckanext/switzerland/harvester/base_sbb_harvester.py [hunk 1]
```diff
@@ -97,7 +97,6 @@ class BaseSBBHarvester(HarvesterBase):
         "accrual_periodicity": "",
         # --------------------------------------------------------------------------
         "description": {"fr": "", "en": "", "de": "", "it": ""},
-        "notes": {"fr": "", "en": "", "de": "", "it": ""},
```
Comment (bellisk) [inline ckanext/switzerland/harvester/base_sbb_harvester.py:100]: We never want a value here - it's a CKAN default field, not in our schema.

File: ckanext/switzerland/plugin.py [hunk 2]
```diff
@@ -156,15 +156,10 @@ def _prepare_package_json(self, pkg_dict):
...
-            # to show the form on resource_edit, so we skip it here
-            path = toolkit.request.path
-            if path.startswith("/api") or toolkit.request.path == "POST":
-                return pkg_dict
-        except TypeError:
-            # we get here if there is no request (i.e. on the command line)
+        # Do not change the resulting dict for API requests and form saves
+        # _package_reduce_to_requested_language removes all translation dicts needed
+        # to show the form on resource_edit, so we skip it here
+        if sh.request_is_api_request() or toolkit.request.path == "POST":
```
Comment (bellisk) [inline ckanext/switzerland/plugin.py:162]: We have a helper method to test if a request comes from the API so let's use it.

File: ckanext/switzerland/harvester/base_sbb_harvester.py (hunk 1 above)
Comment (stefina) [reply ckanext/switzerland/harvester/base_sbb_harvester.py:100]: thank you for explaining!

File: ckanext/switzerland/plugin.py [hunk 3]
```diff
@@ -353,11 +343,6 @@ def after_dataset_show(self, context, pkg_dict):
         # groups
         if pkg_dict["groups"] is not None:
             for group in pkg_dict["groups"]:
-                """
-                TODO: somehow the title is messed up here,
-                but the display_name is okay
-                """
-                group["title"] = group["display_name"]
```
Comment (bellisk) [inline ckanext/switzerland/plugin.py:360]: Solved the 9-year-old mystery! XD

**Review by stefina — APPROVED** [review]

**Pull request merged**